package.json

```diff
@@ -9,14 +9,16 @@
     "email": "team@oxen.io"
   },
   "scripts": {
+    "clean": "rimraf .cache && rimraf build",
```

Comment:
you can just have 
`rimraf .cache build`
that will avoid keeping the build, when cache is empty

Reply:
Fixed!